docker-compose.yml

```diff
@@ -7,9 +7,13 @@ services:
       - PUID=1000
       - PGID=1000
       - TZ=Etc/UTC
+      - USER=prem
+      - FILE__PASS=/transmission_password
     volumes:
       - data-config:/config
-      - data-downloads:/downloads
+      - /mnt/HC_Volume_37156271/from-hf-to-torrent/data:/downloads
```

Comment:
Can't we use docker volumes? and tell to Docker to access the external volume?

Reply:
sure, I'll update

Reply:
Using external volume would create a requirement before running this compose file that the volume should already be created and present. Instead we can create and mount these volumes at runtime of compose file (using type `local`), that sounds fine?

Reply:
Yep